### ISSUE TITLE
[WIP] Using Zoom: Process for Community Forum and Onboarding Calls

### ISSUE THREAD

**Issue by aleesteele** [the-turing-way/the-turing-way#3594]
### Summary

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### What needs to be done?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*

### Who can help?

* *Lorem ipsum dolor sit amet, consectetur adipiscing.*
* *Lorem ipsum dolor sit amet, consectetur adipiscing.*

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

**Comment by aleesteele** [comment]
Notes from discussion with @vhellon:
- Possible to add a custom banner to email confirmation to keep branding similar
- Make sure everything is sent through zoom platform (may be responsible for some of the zoom)
-

**Comment by aleesteele** [comment]
Re-opening this issue after discussion about June 2024 Book Dash - as used for the Community Share-outs & Fireside Chat.

**Comment by da5nsy** [comment]
I've used Indico recently (for Digital Research Academy events) and I've been really happy with it. 

The UI isn't the most intuitive or slick, but it works and I haven't found any bugs or glitches in it, and it has very broad functionality (I haven't found anything that I wanted to do that wasn't available, and I've found many extra bells and whistles that I hadn't even considered).

CCing @HeidiSeibold as she set it up and hosts it and so might have additional thoughts.

**Comment by bbartholdy** [comment]
I've used Eventbrite quite a lot and it generally works fine. They offer code (javascript) to embed the event registration in websites. The main limitation is that a free account only allows you to create events for up to 25 people.

**Comment by RichardJActon** [comment]
In the spirit moving to more open tools when the opportunity presents I would suggest considering [pretix](https://pretix.eu/about/en/) as I think they will have the featureset required. I would say that their licensing approach falls under the slightly controversial [fair-code](https://faircode.io/) category where it is mostly open but violates freedom zero / the 'no discrimination against fields of endeavor' clauses of the free software / open source standard definitions in that that is prohibits uses which directly commercially compete with their Saas offering. You can self-host it but not host an instance which you offer commercial use of to third parties, otherwise it is largely AGPL and Apache 2.0 terms.

Some fully open options:
- [mobilizon](https://mobilizon.org/en/) published by framasoft known to this community already through our use of framapad.
- [gathio](https://gath.io/) which uses the activitypub protocol to federate instances (the same underlying technology as mastodon for those unfamiliar) but they are relatively new and not very feature rich as of yet.

**Comment by HeidiSeibold** [comment]
I really enjoy working with indico and I also like it better than pretix, which is ok, but I don't like the wording ("tickets") in this context as much.

Indico is developed by CERN and used by organisations such as Unesco, so I think it's fair to say that it will stick around. 

I'd be happy to discuss sharing the Digital Research Academy instance, if that is an option for you https://events.digital-research.academy/
💜

**Comment by penyuan** [comment]
I'm personally very curious about mobilizon mentioned by @RichardJActon and Indico by @HeidiSeibold. On top of those, I'm aware of two more options: 

1. [Eventyay](https://eventyay.com) - Fully open source ([repo link](https://github.com/fossasia/open-event-server)), self-hostable and with official instance that you can just pick up and use. I know groups who successfully used Eventyay for organising big hybrid conferences such as the *huge* annual [FOSSASIA](https://eventyay.com/e/55d2a466). In fact I think some FOSSASIA organisers are also the Eventyay developers. 
2. [Ticketpass](https://ticketpass.org) - Claims to be an Eventbrite competitor that "donate 50% of our booking fee to charity and social causes". Out of principle, I cannot officially endorse this option because it is not open source, but listing it here for completeness!

**Comment by Lextuga007** [comment]
I've tried out Eventyay for events for https://github.com/nhs-r-community based on the suggestions here but found it was a little buggy. I couldn't get the tickets to show at first and had to set up a new event in the end, but now it logs me out without warning - used on Chrome and Edge. I'd be really interested to see it in action though as it looks potentially really good as it offers payment through PayPal, Strike and invoices (which is the main way we do payments in the Public Sector).

**Comment by Lextuga007** [comment]
To update I had [Pretix](https://pretix.eu/about/en/) suggested and have tested this out and have been very impressed. It's free for free events, open source and can manage quite complicated situations like a coffee and coding running on different days in different weeks. It's also possible to set up "season tickets" which we need for NHS-R Community as we have restricted event numbers for workshops so can only offer those to colleagues from certain organisations (public sector as an example). I'm currently going through my organisation's Information Governance and Cyber security for approval to use this platform and can share the outcomes and agreements once that's sorted.

**Comment by aleesteele** [comment]
Hi @Lextuga007, @penyuan, @HeidiSeibold, @RichardJActon, @bbartholdy, @da5nsy - thank you all again for your thoughtful input on this issue!

During this time of transition for _The Turing Way_, we are currently using Zoom to host our meetings, and Google forms for collecting sign-ups to community events. While neither of these platforms are open source, we have been using them as interim tools while we develop a longer-term solution and plan for the community this year. We hope to transition to tools like Indico, Pretix, Eventyay etc when we're more stable.

With the @the-turing-way/community-management-working-group - we'll be revisiting this issue and infrastructure for event organising over the next few months. @LelleDi kindly added it to the group's project board.

Thanks everyone! 🙏

**Comment by SaraVilla** [comment]
@JimMadge shall this be a discussion? shall I convert it?

**Comment by JimMadge** [comment]
> [@JimMadge](https://github.com/JimMadge) shall this be a discussion? shall I convert it?

Yes, feels like a discussion. It might be stale or solved though :eyes:.

**Comment by RichardJActon** [comment]
feels pretty discussion-y to me - it's a question that might come up again if there are any issues with current tools.

**Comment by SaraVilla** [comment]
Moving it to discussion and we can revisit whether closing it or action something else in some time
